github/Requester.py

```diff
@@ -413,8 +421,8 @@ def __makeAbsoluteUrl(self, url):
             url = self.__prefix + url
         else:
             o = urlparse.urlparse(url)
-            assert o.hostname in [self.__hostname, "uploads.github.com"], o.hostname
-            assert o.path.startswith((self.__prefix, "/api/uploads"))
+            assert o.hostname in [self.__hostname, "uploads.github.com", "status.github.com"], o.hostname
+            assert o.path.startswith((self.__prefix, "/api/"))
```

Comment:
We change this to `/api/` so that the path can match the patterns of both the upload and status API